src/web/modules/webhooks/webhooks.http.ts

```diff
@@ -12,17 +13,24 @@ if (common.development) {
 
 export async function list(req: Request, res: Response, next: NextFunction): Promise<void> {
   try {
-    const serviceId = req.params.serviceId
-
-    const response = await Webhooks.webhooks.list({
-      service_id: serviceId as string
+    const accountId = req.query.account
+
+    if (!accountId) {
+      throw new Error('Webhooks can currently only be shown for a single account')
+    }
+
+    const account = await Connector.accounts.retrieveAPI(accountId as string)
+    const service = await AdminUsers.services.retrieve({gatewayAccountId: accountId as string})
```

Comment:
I would prefer to use the `service_id` on the gateway account rather than looking up the service by internal gateway account ID -- this will minimise the things we need to change when we stop using the internal gateway account ID.

It is there! It would just need to be added to the types file because I think its missing from that.

Reply:
Ah - thanks, I thought it was missing.
I will update this.